### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by rachely3n**
Created a change log file. Release 1.3.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  
+ ### Changed
+				
+ ### Removed
+				 
+ ### Fixed
+
+## [1.3.0] - 2019-05-24
+
+### Security
```
Comment (stephenashank) [inline CHANGELOG.md]: Remove unused section

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ 
+ ### Added
+  
+ ### Changed
+				
+ ### Removed
+				 
+ ### Fixed
+
+## [1.3.0] - 2019-05-24
```
Comment (stephenashank) [inline CHANGELOG.md]: 2019-06-24

Comment (rachely3n) [reply CHANGELOG.md]: hahaha...

File: CHANGELOG.md (hunk 1 above)
Comment (rachely3n) [reply CHANGELOG.md]: Done

**Pull request merged**